complete/src/main/resources/application.properties

```diff
@@ -1,3 +1,4 @@
 spring.profiles.active=local
 server.port=8080
-cognito.jkws.url=https://cognito-idp.ap-southeast-1.amazonaws.com/ap-southeast-1_iQdl5AVrA/.well-known/jwks.json
\ No newline at end of file
+cognito.jkws.url=https://cognito-idp.ap-southeast-1.amazonaws.com/ap-southeast-1_iQdl5AVrA/.well-known/jwks.json
+sms.enabled=false
```

Comment:
You have defined and set the property here. Now we should have a if condition in the [SmsManager](https://github.com/SimplQ/simplQ-backend/blob/master/complete/src/main/java/com/example/restservice/service/smsService/SmsManager.java) to send sms only if this property is set to true. Can you make this change as well?

Reply:
I have used @ConfigurationProperties to load property into SmsManager